docs/core/porting/windows-compat-pack.md

```diff
@@ -0,0 +1,99 @@
+---
+title: Porting to .NET Core - Using the Windows Compatibility Pack
+description: Porting to .NET Core - Using the Windows Compatibility Pack
+keywords: .NET, .NET Core, Windows, Compatibility
+author: immol
+ms.author: mairaw
+ms.date: 11/13/2017
+ms.topic: article
+ms.prod: .net-core
+ms.devlang: dotnet
+ms.assetid: ffa28aed-45dd-4b02-965f-768c467829c3
+---
+
+# Using the Windows Compatibility Pack
+
+Many customers have reported that the primary reason why they have trouble
+porting to .NET Core is that they depend on APIs and technologies that only
+exist in .NET Framework. The *Windows Compatibility Pack* is about providing a
+good chunk of these technologies so that building .NET Core applications as well
+as .NET Standard libraries becomes much more viable for existing code.
+
+This is a logical [extension of .NET Standard 2.0][API Expansion] which
+significantly increased the API set to ensure code largely already compiles
+as-is. But in order to keep .NET Standard understandable, this didn't include
+technologies that can't work across all platforms, such as registry, WMI, or
+reflection emit APIs.
+
+The *Windows Compatibility Pack* sits above .NET Standard and is thus free to
+provide access to technologies that are Windows only. It is especially useful
+for customers that want to move to .NET Core but plan to stay on Windows as a
+first step. In that scenario, not being able to use Windows-only technologies is
+only a migration hurdle with zero architectural benefits.
+
+## Contents
+
+The *Windows Compatibility Pack* is provided via the NuGet Package
+[Microsoft.Windows.Compatibility] and can be referenced from .NET Core as well
+as .NET Standard.
+
+It provides about 20,000 APIs, including Windows-only as well as cross-platform
+APIs:
+
+* ACLs
+* Code Pages
+* CodeDom
+* Configuration
+* Crypto
+* DirectoryServices
+* Drawing
+* EventLog
+* MEF
+* Odbc
+* Perf Counters
+* Permissions
+* Ports
+* Registry
+* Runtime Caching
+* WCF
+* Windows Services
+
+For more details, take a look at the [spec of the compatibility pack][proposal].
+
+## Getting Started
+
+1. When porting existing code to .NET Core or .NET Standard, install the NuGet
+   package [Microsoft.Windows.Compatibility].
+2. If you want to stay on Windows, you're all set.
+3. If you want to run the .NET Core application or .NET Standard library on
+   Linux or macOS, use the [API Analyzer] to find usage of APIs that won't
+   work cross-platform.
+4. Either remove the usages of those APIs, replace them with cross-platform
+   alternatives, or guard them using a platform check, like:
+    ```CSharp
```

Comment:
The correct identifier is lowercase csharp

Reply:
GitHub doesn't seem to care, but I did indeed change in Sublime. Fixed.

---
In reply to: [150916744](https://github.com/dotnet/docs/pull/3674#discussion_r150916744) [](ancestors = 150916744)

Reply:
Yep, but docs seem to:
![image](https://user-images.githubusercontent.com/12971179/32800439-2bb8ec98-c92f-11e7-8c80-ec0e278db10c.png)

The label should read C# not CSharp.